src/pages/ExplorerPage/ExplorerPage.tsx

```diff
@@ -35,42 +36,87 @@ const ExplorerPage = () => {
 
   const classes = useStyles();
   const [active, setActive] = useState(false);
+  // NOTE: we are going to use the setter upon filters implementation
   const [queryParams, setQueryParams] = useState({ page: "1", limit: "10" });
 
-  const [state, setState] = useState<{ transfers: Transfer[] | never[], loading: "none" | "loading" | "done", isReady: boolean }>({
+  const [state, setState] = useState<{
+    transfers: Transfer[] | never[];
```

Comment:
Type definition can be simplified here to just `Transfer[]`.
```suggestion
    transfers: Transfer[];
```

Reply:
fixed